Makefile

```diff
@@ -102,8 +102,11 @@ build-grpc-interop:
 	@cp -f bazel-bin/external/com_github_grpc_grpc/test/cpp/interop/interop_server bin/
 	@cp -f bazel-bin/external/com_github_grpc_grpc/test/cpp/interop/stress_test bin/
 
-# This target is to upload e2e client binaries so once they get updated,
-# this target should be executed.
+# This target is to upload e2e client binaries so once they get updated.
+# The execution environment of prow image may be imcompatiable with your local
+# builidng environemnt which fails the client binaries so you may have to make
+# this target in the prow image, for which you can find the latest version under
+# https://github.com/GoogleCloudPlatform/oss-test-infra/blob/master/prow/prowjobs/GoogleCloudPlatform/esp-v2/esp-v2.yaml
```

Comment:
hmm,  is this statement correct?  I thought we are upload the client binaries into GCS bucket.  I assume our test scripts download it from the GCS bucket.

Reply:
Our build job currently doesn't build and uploads client binaries, which is fine since it is not modified often(I guess it hasn't for half year).

Reply:
So the comment should be
# E2e tests are using e2e-client-binaries from GCS bucket, if they are updated, make this target
# to upload the binaries to GCS bucket

Reply:
Actually, I am trying to emphasize the library incompatility of OS.

If you want, I can also add this piece.

Reply:
Done.